src/test/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentServiceSpec.scala

```diff
@@ -12,8 +12,13 @@ import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
 import uk.gov.nationalarchives.Tables.{BodyRow, ConsignmentRow, ConsignmentstatusRow, SeriesRow}
 import uk.gov.nationalarchives.tdr.api.db.repository._
-import uk.gov.nationalarchives.tdr.api.graphql.fields.ConsignmentFields.{AddConsignmentInput, FileChecks, StartUploadInput,
-  UpdateConsignmentSeriesIdInput, UpdateExportDataInput}
+import uk.gov.nationalarchives.tdr.api.graphql.fields.ConsignmentFields.{
+  AddConsignmentInput,
+  FileChecks,
+  StartUploadInput,
+  UpdateConsignmentSeriesIdInput,
+  UpdateExportDataInput
+}
```

Comment:
I guess this is scalastyle complaining about the length of the line. We don't have many of these multi line imports so would it be better to just do `import uk.gov.nationalarchives.tdr.api.graphql.fields.ConsignmentFields._`